deploy/scripts/src/common/utils/config/load-profiles.ts

```diff
@@ -93,7 +93,8 @@ export enum LoadProfile {
   smoke,
   short,
   full,
-  deployment
+  deployment,
+  extendedRampUp
```

Comment:
Perhaps call rename this to `rampOnly`? This currently makes it seem like it's a longer ramp up than usual which isn't the case